com.mattermost.Desktop.appdata.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.mattermost.Desktop</id>
+  <name>Mattermost</name>
+  <project_license>Apache-2.0</project_license>
+  <developer_name>Mattermost, Inc.</developer_name>
+  <summary>Opensource team chat</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://www.mattermost.org/</url>
+  <description>
+	<p>
+       As an alternative to proprietary SaaS messaging, Mattermost brings
+       all your team communication into one place, making it searchable and
+       accessible anywhere. It’s written in Golang and React and runs as
+       a production-ready Linux binary under an MIT license with either MySQL
+       or Postgres.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://ga0.imgix.net/screenshot/o/111417-mattermost-1513705902-5351114</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://ga0.imgix.net/screenshot/o/111417-mattermost-1513706072-6569283</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://ga0.imgix.net/screenshot/o/111417-mattermost-1513705838-1315444</image>
+    </screenshot>
+  </screenshots>
+  <releases>
+    <release version="4.1.2" date="2018-05-25"/>
+  </releases>
+  <content_rating type="oars-1.1">
```

Comment:
If all is `none` this can be an empty tag: https://hughsie.github.io/oars/generate.html

Reply:
nice, thanks!